resources/views/site/staff.blade.php

```diff
@@ -60,10 +60,11 @@ class="img-responsive center-block profile-picture"/>
                     </div>
                     <div class="panel-body">
                         <h4>Operations Director (VATUK9)</h4><br/>
-                        <img src="{{ $teamPhotos[54] }}"
+                        <p class="text-left">Vacant</p>
+                        <!--<img src="$teamPhotos[54]"
                              width=50px
                              class="img-responsive center-block profile-picture"/>
-                        <p class="text-center">Chris Pawley</p>
+                        <p class="text-center">Chris Pawley</p>-->
```

Comment:
These lives could be removed rather than commented out for clarity and ease of reading

Reply:
Wasn't sure what to do, thanks for the clarification!